content/blog/2022/09/vimdap/index.md

```diff
@@ -0,0 +1,189 @@
+---
+title: "Configure Camel Debug Adapter in Vim to enable Camel Textual Route debugging"
```

Comment:
as a title, I'm wondering if we cannot make it shorter and emphasize the important part fo rCamel users. Something like: "Camel Textual Route debugging in Vim"
Your title is fine too, just tryign to give more ideas